src/SiA.Library/SiA.Library.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net48;netcoreapp3.0</TargetFrameworks>
```

Comment:
It's better to try to target .NET Standard 2.0 so it works with any runtime.

Reply:
Also copy all the properties that are in the csproj like `<Copyright>`, `license` and so.